camel-k-runtime-core/src/main/java/org/apache/camel/k/Sources.java

```diff
@@ -18,177 +18,119 @@
 
 import java.io.ByteArrayInputStream;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Base64;
 import java.util.Collections;
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.zip.GZIPInputStream;
 
 import org.apache.camel.CamelContext;
 import org.apache.camel.k.support.StringSupport;
```

Comment:
I wonder if some of the StringSupport you have here can use what we have in camel-core, and if not maybe move those to camel-util etc.

Reply:
yeah that's something we have been discussing yesterday, my plan is to move whatever generic I have here to camel